eng/common/scripts/Package-Properties.ps1

```diff
@@ -10,12 +9,12 @@ class PackageProps
     [string]$pkgChangeLogPath
     [string]$pkgGroup
 
-    PackageProps([string]$pkgName,[string]$pkgVersion,[string]$pkgDirectoryPath,[string]$pkgServiceName)
+    PackageProps([string]$pkgName, [string]$pkgVersion, [string]$pkgDirectoryPath, [string]$pkgServiceName)
```

Comment:
I am little confused here. overloaded constructor also will do the same since you have defaulted pkggroup. right?

Reply:
That what you'll think. But from various testing I did PowerShell does not support constructor overloading or chaining which is why the implementation looks weird. If you know a cleaner way to go about it please share with me.